components/net_traits/bluetooth_scanfilter.rs

```diff
@@ -0,0 +1,132 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use device::bluetooth::BluetoothDevice;
+use std::collections::HashSet;
+use std::iter::FromIterator;
+
+const MAX_DEVICE_NAME_LENGTH: usize = 29;
+
+#[derive(Deserialize, Serialize)]
+pub struct ServiceUUIDSequence(Vec<String>);
+
+impl ServiceUUIDSequence {
+
+    pub fn new(vec: Vec<String>) -> ServiceUUIDSequence {
+        ServiceUUIDSequence(vec)
+    }
+
+    fn is_empty(&self) -> bool {
+        self.0.is_empty()
```

Comment:
it returns the first and only element (Vec<String>) of the tupple struct: ServiceUUIDSequence(Vec<String>)